tests/test_anemoi.py

```diff
@@ -0,0 +1,77 @@
+import sys
+from pathlib import Path
+import numpy as np
+from torch.utils.data import DataLoader
+
+# Add the project root to Python path so we can import from graph_weather
+project_root = Path(__file__).parent.parent
+sys.path.insert(0, str(project_root))
+
+from graph_weather.data import AnemoiDataset
+
+
+def test_anemoi_dataset():
+    """Test the AnemoiDataset class with real or fixture data"""
+    dataset_config = {
+        "dataset_name": "era5-test",
```

Comment:
Just for my clarification, is `era5-test` a test dataset included in Anemoi? Where is the data coming from?